systemFlake.nix

```diff
@@ -62,8 +64,8 @@ let
     "hosts"
     "hostDefaults"
     "nixosProfiles"
-    "channels"
-    "channelsConfig"
+    "streams"
+    "streamsConfig"
```

Comment:
old `channels` and `channelsConfig` should not be removed for backward compatibility

Reply:
forced pushed a fix, is that enough to do it?